sdk/storage/azure-storage-file-datalake/tests/perfstress_tests/read.py

```diff
@@ -0,0 +1,36 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure_devtools.perfstress_tests import get_random_bytes, WriteStream
+
+from ._test_base import _FileSystemTest
+
+
+class DownloadTest(_FileSystemTest):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        file_name = "downloadtest"
+        self.file_client = self.fs_client.get_file_client(file_name)
+        self.async_file_client = self.async_fs_client.get_file_client(file_name)
+
+    async def global_setup(self):
+        await super().global_setup()
+        data = get_random_bytes(self.args.size)
+        await self.async_file_client.create_file()
+        await self.async_file_client.upload_data(data, overwrite=True)
+
+    def run_sync(self):
+        download = WriteStream()
```

Comment:
Can WriteStream be reused?

Reply:
No really, as it maintains positional state - which in turn is used by the SDK for multi-threaded downloads